README.md

```diff
@@ -1,320 +1,66 @@
 
-### Full Node Configuration
-1. Initialize the full node
-```sh
-celestia full init --core.remote <ip:port of celestia-app> --headers.trusted-hash <hash_from_celestia_app>
-```
+### Celestia Full Node
+When we are mentioning `Celestia Full Node`, we are relating to a combination of 2 components: 
 
-Example:
-```sh 
-celestia full init --core.remote tcp://127.0.0.1:26657 --headers.trusted-hash 3BDFBD7E2D97215CFA600DD8B39AAEECC015E43FEE7B8A4D8A8B630E8B4D4006
-```
+- Celestia Application (lives in [celestia-app](https://github.com/celestiaorg/celestia-app) repo)
+- Celestia Full Node (lives in [celestia-node](https://github.com/celestiaorg/celestia-node) repo)
 
-2. Edit configurations
+<i>Conceptually</i>, one can't live without another as we need every new block to be erasure coded in order to do data availability sampling from `Celestia Light Clients`
 
-If you have multiple celestia full nodes, then you need to add them as mutual peers in the `config.toml` file and allow the peer exchange. This is needed in order for celestia full nodes communication between each other.
-```sh
-nano ~/.celestia-full/config.toml
-```
-```sh
-...
-[P2P]
-  ...
-  MutualPeers = ["/ip4/<ip>/tcp/2121/p2p/<pubKey>"] #add multiaddresses of other celestia full nodes
-  PeerExchange = true #change this line to true. Be default it's false
-  ...
-...
-```
+Technically, Celestia Application and Celestia Full Node are operating on different processes of <b>one instance</b>(e.g. server). Let's break down what each of the parties do. 
 
-3. Start the full node
-```sh
-celestia full start
-```
-Now, the celestia full node will start syncing headers and storing blocks from celestia app. 
+Celestia Application(CA) takes care of these points: 
+- Consensus
+- State Interaction
+- Block Production
+- Communication between Celestia Application instances only!
 
-<u>Note: At startup, we can see the `multiaddress` from celestia full node. This is <b>needed for future light client</b> connections and communication between celestia full nodes</u>
+Celestia Full Node(CFN) takes care of these points: 
+- Sync blocks from Celestia App
+- Erasure Codes the blocks
+- Serve headers/shares to Celestia Light Clients
+- Communication between Celestia Full Nodes and Celestia Light Clients
 
-Example:
-```sh
-/ip4/46.101.245.50/tcp/2122/p2p/12D3KooWKNBZvF93L92aTYs6jRyozicGmuu3cF9gotMtUCeHAPYN
-```
+Communication of CA and CFN processes happens over the RPC.
 
-### Light Client Configuration
-To start the light client, we need to know 2 variables:
-- Trusted peer’s multi address to connect to (a celestia full node is the case here)
-- Trusted block hash from celestia-app
+### Celestia Light Client
+Celestia Light Client(CLC) lives only in [celestia-node](https://github.com/celestiaorg/celestia-node) repo. What does CLC do: 
```

Comment:
Should we call this a "Celestia Light Node"

Reply:
On it